File: api/network/public/pas/network.pas
Unit network;

Interface

procedure findRoute (N, a, b : integer);

Implementation

uses graderlib;

procedure findRoute (N, a, b : integer);
var
    i : integer;
begin
    {
        Obviously, this is not a good solution.
        Replace it with your own code.
    }
    for i := 1 to ping (a, b) do
        travelTo (i);
end;

End.

File: api/network/public/cpp/network.cpp
#include "network.h"

void findRoute (int N, int a, int b)
{
    /*
     *  Obviously, this is not a good solution.
     *  Replace it with your own code.
     */

    int dist = ping(a, b);
    for (int i = 1; i <= dist; ++i)
        travelTo (i);
}
